dotnet/Workloads/vs-workload.template.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <PropertyGroup>
-    <TargetName>Microsoft.NET.Sdk.MaciOS.Workload</TargetName>
+    <TargetName>Microsoft.NET.Sdk.MaciOS.Workload.@VSMAN_VERSION@</TargetName>
```

Comment:
This looks ok, if we like the name `Microsoft.NET.Sdk.MaciOS.Workload.7.0`.

Reply:
Updated to `Microsoft.NET.Sdk.MaciOS.Workload.net7.0` to make the 7.0 less arbitrary.